api/src/main/java/org/openmrs/module/commonlabtest/LabTest.java

```diff
@@ -220,6 +220,28 @@ public String serialize() {
 		}
 	}
 
+	/**
+	 * Returns the first non-voided commonlabtest attribute matching a commonlabtest attribute type.
+	 * <br>
+	 * <br>
+	 * Returns null if this commonlabtest has no non-voided {@link LabTestAttribute} with the given
+	 * {@link LabTestAttributeType}, the given {@link LabTestAttributeType} is null, or this
+	 * commonlabtest has no attributes.
+	 * 
+	 * @param lat the LabTestAttributeType to look for {@link LabTestAttributeType#equals(Object)}
+	 * @return LabTestAttribute that matches the given type
+	 */
+	public LabTestAttribute getAttribute(LabTestAttributeType lat) {

```

Comment:
Change the parameter name to fully qualified name.

Reply:
Something similar in  OpenMRS core:

https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Person.java
-- > public PersonAttribute getAttribute(PersonAttributeType pat)